### PR TITLE
Initial content for the IGTF role

### PR DESCRIPTION
**Pull request by orviz**

### CODE REVIEW

**Review by gwarf — CHANGES_REQUESTED** [review]
Hi Pablo, thanks!
So the molecule tests are failing: https://travis-ci.org/EGI-Foundation/ansible-role-igtf/builds/603852409?utm_source=github_status&utm_medium=notification
You can find some doc in the Ansible Style Guide that @brucellino created: https://github.com/EGI-Foundation/ansible-style-guide
And en example repo using molecule is https://github.com/EGI-Foundation/ansible-role-umd

Please let me know if you need some support/can implement this, would be cleaner to have it before merging.

**Comment by orviz** [comment]
@gwarf I put there the CA/CRL tests from EGI-Foundation/ansible-role-umd. When merged, we should remove them from that role

File: tasks/crl.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- name: Remove repository
+  yum_repository:
+    name: epel
+    enabled: no
+    description: 'Extra Packages for Enterprise Linux 7 - $basearch'
+    metalink: https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch&infra=$infra&content=$contentdir
+  notify: yum-clean-metadata
+  when: (ansible_os_family | lower) == 'redhat'
+
```
Comment (gwarf) [inline tasks/crl.yml:23]:
```suggestion
```

Comment (gwarf) [reply tasks/crl.yml:23]: Just to find something :)
Please commit the change then I will merge this PR, thanks!

Comment (orviz) [reply tasks/crl.yml:23]: @gwarf done!

**Review by gwarf — APPROVED** [review]
LGTM, 👍 
Thanks Pablo!

**Pull request merged**